test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -1150,7 +1152,9 @@ public void testInitVerifyColumnNames_uniqueKey_updateSql() throws Exception {
   }
 
   @Test
-  public void testInitVerifyColumnNames_singleDocContentSql() throws Exception {
+  public void
+      testInitVerifyColumnNames_singleDocContentSql_uniqueKeyNotInSelect()
```

Comment:
I don't think that is better, and perhaps even worse, since it sounds like it should be an error.